bunq/sdk/exception_factory.py

```diff
@@ -21,40 +21,86 @@ class ExceptionFactory:
 
     # Constants for formatting messages
     _FORMAT_RESPONSE_CODE_LINE = 'HTTP Response Code: {}'
-    _GLUE_ERROR_MESSAGES = '\n'
+    _FORMAT_RESPONSE_ID_LINE = 'The response id to help bunq debug: {}'
+    _FORMAT_ERROR_MESSAGE_LINE = 'Error message: {}'
+    _GLUE_ERROR_MESSAGE_NEW_LINE = '\n'
+    _GLUE_ERROR_MESSAGE_STRING_EMPTY = ''
 
     @classmethod
-    def create_exception_for_response(cls, response_code, messages):
+    def create_exception_for_response(
+            cls,
+            response_code,
+            messages,
+            response_id
+    ):
         """
+        :type response_id: str
         :type response_code: int
         :type messages: list[str]
 
         :return: The exception according to the status code.
         :rtype:  ApiException
         """
 
-        error_message = cls._generate_message_error(response_code, messages)
+        error_message = cls._generate_message_error(
+            response_code,
+            messages,
+            response_id
+        )
 
         if response_code == cls._HTTP_RESPONSE_CODE_BAD_REQUEST:
```

Comment:
Consider creating a mapping (response_code => exception type) for all these cases

Reply:
This code has not been modified in this pr, please create a follow up issue for this. 

If so, this must also be refactored in the other SDK's